### DISCUSSION TITLE
Removal of tags broke pre-commit configs

### DISCUSSION THREAD

**Discussion by danielpof** [commitizen-tools/commitizen#931] [General]
Hi, just wanted to let the project maintainers know that the removal of tags (such as 3.4.0) broke all of our pre-commit configs (which used it as a reference).

I've already resolved the problem for my organization by doing a mass update to all of our repos/projects but hope such breaking changes in the future take in consideration the effort hooks users will have to expend.

**Comment by woile** [comment]
Sorry for the inconvenience, which PR broke your pipeline? Could you provide a bit more information? Ideally so we can add tests

**Reply by danielpof** [reply]
It wasn't a PR but the removal of the tag '3.4.0' from the repo.

 `.pre-commit-config.yaml` references a tag to pull the hooks, in our case we had `3.4.0` and other removed tags, additionally we had pre-commit as a PR validation step.

So the removal of the tag made pre-commit unable to download the hooks making our pre-commit pipeline step fail.

**Reply by woile** [reply]
I might be lost here, but we didn't remove any tags. I can see tag `v3.4.0` in our repo 🤔

**Reply by danielpof** [reply]
We had been using `pre-commit autoupdate` and had several repos using the tag without the `v` now those tags no longer exist. Some examples are `3.4.0`, `3.2.2`, etc

I can only assume the tags without `v` were removed as there no history of deletions.
